aigpk docs (#2100)
ngrok/ngrok-docs: ai-gateway/guides/configuring-providers.mdx

---
title: Configuring Providers
description: How to configure AI providers, custom endpoints, and advanced key selection in Traffic Policy.
---

> This guide covers Traffic Policy provider configuration. This is required for custom or self-hosted providers. For standard providers (OpenAI, Anthropic, Google, etc.), attach your keys directly to your AI Gateway API Key instead—see [Attaching Provider Keys](/ai-gateway/guides/attaching-provider-keys).

Reviewer: nits:
- avoid latinisms
- when the body text describes what you'll find on the other side of the link, you can just use that as the anchor text to avoid being redundant

```suggestion
This guide covers Traffic Policy provider configuration. This is required for custom or self-hosted providers. For standard providers (such as OpenAI, Anthropic, and Google), [attach your keys directly](/ai-gateway/guides/attaching-provider-keys) to your AI Gateway API Key instead.
```
